docs/FEATURES.md

```diff
@@ -425,6 +425,19 @@ tasks:
     namespace: carts
 ```
 
+### Specify annotations for Job
```

Comment:
It would be nice if the Headline is listed in the index on the top of the FEATURES files